core/sql/sqlcomp/PrivMgrMD.cpp

```diff
@@ -151,21 +151,19 @@ PrivStatus PrivMgrMDAdmin::initializeComponentPrivileges()
 
 {
 
-// First, let's start with a clean slate.  Drop all components as well as 
-// their respective operations and and any privileges granted.  This should be  
-// a NOP unless PrivMgr metadata was damaged and reintialization is occurring.
-
-PrivMgrComponents components(metadataLocation_,pDiags_);
-
-   components.dropAll();
-   
 // Next, register the component.
 
 PrivStatus privStatus = STATUS_GOOD;
 
+   // First, let's start with a clean slate.  Drop all components as well as 
```

Comment:
Comments need a little wordsmithing perhaps. (We have "First" coming after "Next.)